docs/pages/datapoint.js

```diff
@@ -39,9 +39,70 @@ export default function DatapointPage({ generatedDocGen }: {| generatedDocGen: D
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Whenever possible, use the full number with separators to ensure clarity across all languages."
+            defaultCode={`
+<Datapoint size="lg" title="Total impressions" value="1,451" trend={{value: 10.1, accessibilityLabel: "Trending up"}}  />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use long decimal values for trend. Limit the trend to one decimal point."
+            defaultCode={`
+<Datapoint size="lg" title="Saves" value="10,392" trend={{value: -12.193, accessibilityLabel: "Trending down"}} />
+`}
+          />
+          {/*
+            The example below should also display a localized trend value, but given trend accepts only a number,
+            the value cannot be localized. Once the API is changed, we should add a localized trend to this example.
+            */}
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Make sure to localize numeric values."
```

Comment:
Would this one be better at the Localization section? 

I don't know if I see it as a best practice.

Reply:
I added this specifically because we typically think of localization around language as opposed to numbers. Happy to remove down the road when we're able to link to more detailed localization docs.

Reply:
Got it.

Reply:
I think I'd still repeat the same concept on the Location section. Kinda using the same words you just used, "We typically think of localization around language as opposed to numbers. In Datapoint, don't forget to localize the numbers as well.  

My main question is, how do we support ,  if we do, number localization in Pinboard?